protocol/common/message.go

```diff
@@ -161,6 +162,7 @@ type Message struct {
 }
 
 func (m *Message) PrepareImageURL(port int) {
+	m.QuotedMessage.PrepareImageURL(port)
```

Comment:
Maybe nil checking?

Reply:
this might crash I think otherwise

Reply:
I had a look at that, but this function is only called twice in both instances it takes its params from `m.imageServer.Port`.

PORT is defined on the Server struct like below:

```go
type Server struct {
	Port   int
	run    bool
	server *http.Server
	logger *zap.Logger
	db     *sql.DB
	cert   *tls.Certificate
}
```

A completely uninstantiated struct would still return 0, see below:

```go
func Test(t *testing.T) {
	type Server struct {
		Port   int
		run    bool
	}

	var s Server

	spew.Dump(s)
}
```

outputs

```
=== RUN   Test
(accounts.Server) {
 Port: (int) 0,
 run: (bool) false
}
--- PASS: Test (0.00s)
PASS
```

Reply:
I was referring to `QuotedMessage`, since it's a pointer I believe
`m.QuotedMessage.PrepareContent(int)` should panic if `QuotedMessage` is nil (golang is a bit weird with nil pointers, since if the function does not access any state it would not panic, but PrepareContent does access some state `m.ID`)

Reply:
Oh, sorry I see. Yes https://github.com/status-im/status-go/pull/2591/files#r828505600, I think we are on the same page.